flink-java/src/main/java/org/apache/flink/api/java/hadoop/mapreduce/HadoopInputFormatBase.java

```diff
@@ -131,6 +136,11 @@ public BaseStatistics getStatistics(BaseStatistics cachedStats) throws IOExcepti
 			throw new RuntimeException(e);
 		}
 
+		// Configure input format for splits
+		if (mapreduceInputFormat instanceof Configurable) {
```

Comment:
I think this is not necessary, because the JM also calls `configure()` before requesting InputSplits.